### ISSUE TITLE
XDSDocumentEntry.referenceIdList - Regione Emilia-Romagna

### ISSUE THREAD

**Issue by marcodevanna** [ministero-salute/it-fse-support#505]
Buongiorno,
abbiamo un dubbio sulla valorizzazione di questo elemento.

Questo metadato presente nell'Affinity Domain, paragrafo 2.14, serve per riportare, tra gli altri, l'accession number necessario per individuare il link alle immagini radiografiche dei referti, conservate nei PACS aziendali.
Non è tuttavia definita una regola precisa su come questo elemento vada compilato, tenendo conto che per individuare l'immagine, oltre all'accession number, servono anche  AEtitle, patientId. 
Inoltre, è possibile che più Accession Number siano da trasmettere insieme.

E' già stato aperto un issue da vari mesi (https://github.com/ministero-salute/it-fse-support/issues/184) al quale tuttavia non è stata data risposta.

E' opportuno definire regole condivise per la sua compilazione, per evitare scenari in cui ciascun implementatore / Regione fa a modo suo, rendendo l'interoperabilità complessa se non impossibile.

**Comment by TDAEY** [comment]
Buongiorno, chiederei cortesemente di indicare il nome della Regione, da cui proviene la richiesta, all'interno del titolo della issue

**Comment by marcodevanna** [comment]
> Buongiorno, con le nuove specifiche 2.6.2 di Affinity domain risulta superato il dubbio?
> 
> Grazie, Marianna

No, ma dopo alcuni approfondimenti, si può chiudere la issue, perchè la tematica richiede un intervento più strutturale.
